src/Tribe/Aggregator/Tabs/Scheduled.php

```diff
@@ -237,15 +237,14 @@ private function action_run_import( $records = array() ) {
 
 			$child = $record->create_child_record();
 			$status = $child->queue_import();
+			$child->update_meta( 'interactive', true );
 			$child->finalize();
 			$child->process_posts();
 
 			if ( is_wp_error( $status ) ) {
 				$errors[ $record->id ] = $status;
 				continue;
 			}
-
-			//$success[ $record->id ] = $status;
```

Comment:
No, because interactively run schedules get the new polling notice rather than the "1 scheduled import has been queued" message.